### DISCUSSION TITLE
Improvements to SMTP settings

### DISCUSSION THREAD

**Discussion by JennieOcken** [forem/forem#16179] [Completed]
Allow a Creator who sets up their own SMTP settings to specify the `from email address` that will be used with that email service in the existing SMTP settings area.

In addition, update the logic of SMTP settings so cloud hosted Forems can override the SendGrid provisioned for them by us with their own SMTP settings (currently this isn't supported in the app).

```return sendgrid_settings if ENV["SENDGRID_API_KEY"].present?```

**Comment by JennieOcken** [comment]
Converted to [issue 16180](https://github.com/forem/forem/issues/16180)

**Comment by ellativity** [comment]
Spamming the upvote reactions as much as I feasibly can...
